File: pset1/mario.c
#include<stdio.h>
#include<stdlib.h>
#include<cs50.h>

int main(){

    //variable declarations
    int height= 0;
    int spaces = 0;
    int hashes = 0;
    int i=0, j=0, k=0;

    // validate input (No string validation)
    do{
        height = get_int("Enter height: ");
    }while(height < 0 || height > 23 );

    // create mario mountain
    for(i = 1; i <= height ; i++){

        spaces = height - i;
        for(j=0; j<spaces; j++){
            printf(" ");
        }

        hashes = height - spaces + 1;
        for(k=0; k<hashes; k++){
            printf("#");
        }
        printf("\n");
    }
    return 0;
}

File: pset6/mario/mario.py
import cs50

height = 0
while True:
    height = cs50.get_int("Enter height: ")

    if height < 0 or height > 23:
        continue
    else:
        break

for i in range(height):

    spaces = height - i - 1;
    print(" " * spaces, end="")

    hashes = height - spaces
    print("#" * hashes, end="")

    print("  ", end="")

    print("#" * hashes, end="")
    print()
